packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -96,6 +105,156 @@ export async function generateArmTemplate(ctx: SolutionContext): Promise<Result<
   return ok(undefined); // Nothing to return when success
 }
 
+export async function deployArmTemplates(ctx: SolutionContext): Promise<Result<void, FxError>> {
+  const azureInfraDir = path.join(ctx.root, baseFolder);
+
+  // update parameters
+  const parameterTemplate = await fs.readFile(
+    path.join(azureInfraDir, parameterFolder, parameterTemplateFileName),
+    ConstantString.UTF8Encoding
+  );
+  const parameterJson = JSON.parse(expandParameterPlaceholders(ctx, parameterTemplate));
+  const parameterDefaultFilePath = path.join(
+    azureInfraDir,
+    parameterFolder,
+    parameterDefaultFileName
+  );
+  await fs.writeFile(parameterDefaultFilePath, parameterJson);
+  const resourceGroupName = parameterJson.parameters.resourceBaseName;
+  if (!resourceGroupName) {
+    throw returnSystemError(
+      new Error("Failed to get resource group from parameters."),
+      PluginNames.SOLUTION,
+      SolutionError.NoResourceGroupFound
+    );
+  }
+
+  // Compile bicep file to json
+  const orchestrationFilePath = path.join(
+    azureInfraDir,
+    templateFolder,
+    bicepOrchestrationFileName
+  );
+  const armTemplateJsonFilePath = path.join(azureInfraDir, templateFolder, armTemplateJsonFileName);
+  await compileBicepToJson(orchestrationFilePath, armTemplateJsonFilePath);
+  ctx.logProvider?.info("Successfully compile bicep files to JSON.");
```

Comment:
it is better to add a prefix for the log
![image](https://user-images.githubusercontent.com/16380704/127463047-80d980b4-3005-475a-b136-325b8b0df74b.png)

Reply:
Fixed.